infrastructure/tooling/src/changelog/index.js

```diff
@@ -111,13 +127,21 @@ class ChangeLog {
       'minor': '[minor] ',
       'patch': '',
     };
+
+    const silentCount = this.snippets.filter(sn => sn.level === 'silent').length;
+    const changesUrl = `https://github.com/taskcluster/taskcluster/tree/v${await this.next_version()}%5E/changelog`;
+    const silentSuffix = silentCount === 0 ?
+      '' :
+      `\n\n*This release includes additional silent changes; see ${changesUrl} for details.*`;
```

Comment:
"silent changes" is a bit confusing. Maybe just "trivial/other changes" or something like that.

Reply:
I wanted to avoid "trivial" since it .. trivializes contributors contributions :)

Do you mean it's confusing just in this one place?  I could change this to "This release contains additional changes and improvements not listed here."